config-linux.md

```diff
@@ -154,29 +154,48 @@ In addition to any devices configured with this setting, the runtime MUST also s
 ## Control groups
 
 Also known as cgroups, they are used to restrict resource usage for a container and handle device access.
-cgroups provide controls to restrict cpu, memory, IO, pids and network for the container.
+cgroups provide controls (through controllers) to restrict cpu, memory, IO, pids and network for the container.
 For more information, see the [kernel cgroups documentation][cgroup-v1].
 
 The path to the cgroups can be specified in the Spec via `cgroupsPath`.
-`cgroupsPath` is expected to be relative to the cgroups mount point.
-If `cgroupsPath` is not specified, implementations can define the default cgroup path.
+`cgroupsPath` can be used to either control the cgroup hierarchy for containers or to run a new process in an existing container.
+If `cgroupsPath` is:
+* ... an absolute path (starting with `/`), the runtime MUST take the path to be relative to the cgroup mount point.
+* ... a relative path (not starting with `/`), the runtime MUST create a new leaf in the cgroup hierarchy and MAY interpret the path relative to a runtime-determined location in the cgroup hierarchy.
```

Comment:
“MUST create a new leaf” conflicts with the later “…MUST consistently attach to the same place …” (see also [here][1]).  Now that we have the consistently-attach wording, I'm ok with weaker wording like [this][2], and don't really care about [MAY vs. SHOULD][3].

[1]: https://github.com/opencontainers/runtime-spec/pull/493#discussion_r66563235
[2]: https://github.com/opencontainers/runtime-spec/pull/493#discussion_r66475713
[3]: https://github.com/opencontainers/runtime-spec/pull/493#discussion_r66557226